dagobah/core/core.py

```diff
@@ -1100,3 +1164,23 @@ def _serialize(self, include_run_logs=False, strict_json=False):
         if strict_json:
             result = json.loads(json.dumps(result, cls=StrictJSONEncoder))
         return result
+
+
+class JobTask(object):
+    """ Expandable Task that references a job """
+    def __init__(self, parent_job, job_name, task_name):
+        self.parent_job = parent_job
+        self.job_name = job_name
+        self.name = task_name
+
+    def expand(self):
+        """ Expand this JobTask into a list of tasks """
+        return self.parent.parent.get_job(self.job_name).tasks.values()
```

Comment:
woops, that's what I get for not doing unit tests yet. I think I mean `parent_job` (which I changed at some point to make it distinguishable from the `target_job`